### ISSUE TITLE
Issue with Spacer after upgrade to 2.2.0

### ISSUE THREAD

**Issue by m-schaeffler** [node-red/node-red#3384]
### Current Behavior

After a upgrade to 2.2.0 I get error messages for Spacer nodes at a deploy of changes.
![2022-01-27 14_46_41-Clipboard](https://user-images.githubusercontent.com/31366299/151371962-4c2635f4-6c2d-4ca7-96a9-f0a8413477a1.png)

### Expected Behavior

no error message at deploy

### Steps To Reproduce

upgrade to 2.2.0

### Example flow

```
paste your flow here
```

### Environment

- Node-RED version: 2.2.0
- Node.js version: 14.18.3-deb-1nodesource1
- npm version: 6.14.16
- Platform/OS: Raspbian
- Browser: Firefox 96.0.2 (win64)

**Comment by hardillb** [comment]
What is in the Node-RED log? There should be an error message to show why it failed to load.

**Comment by m-schaeffler** [comment]
there is no error message in node-red-log.
But I have ssen that the sizes in the spacers are gone and I cannot set them again:
![2022-01-27 15_00_38-Node-RED _ node-red lan](https://user-images.githubusercontent.com/31366299/151373635-a01ec009-591b-4a91-8acd-906519976964.png)
After changing the value to something, saving and open ing the values again, it is again at "auto".

**Comment by knolleary** [comment]
What version of Node-RED Dashboard are you using?

**Comment by m-schaeffler** [comment]
node-red-dashboard@3.1.5

**Comment by fmarzocca** [comment]
I have no spacers in that Dashboard group, but still have the warning!
<img width="503" alt="Schermata 2022-01-27 alle 14 54 29" src="https://user-images.githubusercontent.com/8530030/151376664-eb2a712a-ce02-4212-a9fb-9e8e4446a175.png">

<img width="261" alt="Schermata 2022-01-27 alle 15 16 22" src="https://user-images.githubusercontent.com/8530030/151376697-cd980585-6f2b-4484-b3a2-19be8e25f912.png">

**Comment by knolleary** [comment]
@fmarzocca there must be some ui_spacer nodes in your configuration for that dialog to show up. The fact they don't appear in the dashboard sidebar is a separate node-red-dashboard issue.

**Comment by m-schaeffler** [comment]
I have seen a very strange behavior:

1. On the configuration n ode pane, I cannot set the size of a spacer node.
2. But at the dashboard pane, I can

**Comment by knolleary** [comment]
@fmarzocca in the Configuration Nodes sidebar. If you want to followup on that part of this, please use the forum thread you started so this issue can focus on why the spacer nodes are being flagged as having an error in 2.2.0

**Comment by fmarzocca** [comment]
Ok, I will followup on the forum, but I have sent the  screenshot of my dashboard group in my previous post.

**Comment by Steve-Mcl** [comment]
@m-schaeffler I have tried quite a few combinations to re-create this - but without success.

I do remember this was an issue some time back: https://discourse.nodered.org/t/spacer-size-blocked-to-1x1/56462 

Did you upgrade dashboard at the same time as node-red? After node-red upgrade? before node-red upgrade?

Are there any errors in the browsers console (dev tools console)?

Are you still getting this error - can you post your flow?

Any hints you can provide would be useful

**Comment by m-schaeffler** [comment]
Browser Console at reload (F5) of NodeRed:

        JQMIGRATE: Migrate is installed, version 3.3.0 vendor.js:4:711
        Source-Map-Fehler: Error: request failed with status 404
        Ressourcen-Adresse: https://node-red.lan/vendor/vendor.js?v=2.2.0
        Source-Map-Adresse: purify.min.js.map
        
        Node-RED: 2.2.0 red.min.js:16:23893
        Versions red.min.js:16:23929
        jQuery 3.5.1 red.min.js:16:23964
        jQuery UI 1.12.1 red.min.js:16:23997
        ACE 1.4.12 red.min.js:16:24047
        MONACO 0.28.1 red.min.js:16:24093
        D3 3.0.3 red.min.js:16:24141
        Node-RED Lint Worker started nrlint-worker.js:1:10409

At deploy no message at all.

**Comment by m-schaeffler** [comment]
I did this

1. In the info pane, I deactivated all spacers
2. => issue gone
3. I activated them again step by step and identified 6 of them causing this issue.
4. after deleting them, the issue was gone.
5. In the git pane I discovered that all spacers that are shown with the flows, are gone. All remaining spacers are only shown in the top "global nodes" group.
![Bildschirmfoto vom 2022-01-27 18-05-12](https://user-images.githubusercontent.com/31366299/151408082-b68f8cb1-2476-4373-8eb4-3449588814fa.png)
